### PR TITLE
chore(website): API Page improvements

### PR DESCRIPTION
**Pull request by sjelin**
Should close https://github.com/angular/protractor/pull/2169

[Review on Reviewable](https://reviewable.io/reviews/angular/protractor/2287)

### CODE REVIEW

File: website/js/api-controller.js [hunk 1]
```diff
@@ -121,8 +121,9 @@
    * Set the extra properties used by the view to show the docs.
    * @param list
    */
+  var itemsByName = {};
```
Comment (juliemr) [inline website/js/api-controller.js]: Make this a property of the controller (`this.itemsByName`)

File: website/js/api-controller.js [hunk 2]
```diff
@@ -230,6 +265,8 @@
           return item.name && item.name.match(nameExpr);
         })
       };
+
+      (itemsByName[parentName] || {}).extension = true;
```
Comment (juliemr) [inline website/js/api-controller.js]: maybe
```
if (itemsByName[parentName]) {
  itemsByName[parentName].extension = true
}
```
it's longer but clearer.

File: website/partials/api.html [hunk 3]
```diff
@@ -32,16 +32,17 @@ <h3 id="title">Protractor API <small>{{::version}}</small></h3>
         <span ng-repeat="item in items | filter:searchTerm"
             ng-switch="item.type">
             <div ng-switch-when="title"
-                class="title"
+                class="title {{item.treeClasses}}"
```
Comment (juliemr) [inline website/partials/api.html]: Optional:
`class="title" ng-class="item.treeClasses"`

I'm not sure which I like better, up to you.

File: website/css/protractor.less [hunk 4]
```diff
@@ -119,6 +138,10 @@
     font-size: 1.1em;
     padding-right: 10px;
   }
+
```
Comment (juliemr) [inline website/css/protractor.less]: Can we just do a non-nested .extension definition once?

**Comment by juliemr** [comment]
Link to the closing issue just links to itself.

**Comment by sjelin** [comment]
All comments addressed

**Comment by sjelin** [comment]
(accidentally closes and re-opened)

**Comment by juliemr** [comment]
Style: I find the indentation somewhat confusing. I think it would be better if the `inherited from` blocks were at the same indentation level as the methods... e.g. (with dots for spaces)

element(locator)
..then
..clone
..<i>Inherited from webdriver.WebElement</i>
..equals
..getDriver
..getId

That makes it clearer to me that `getId` is a method on `element`.

**Comment by juliemr** [comment]
We should do something with the whole `webdriver` section at the bottom (maybe `protractor` and then `inherits from webdrivder`), but I think that should be a separate PR.

**Comment by sjelin** [comment]
Fixed the indentation.  I agree that it's better now.  I donno what you have in mind with the webdriver section at the bottom.  At least it's smaller now :P

**Comment by juliemr** [comment]
LGTM.

**Comment by sjelin** [comment]
Merged in at 9267f45

**Pull request closed without merging**